emergency_behaviours/scripts/nogo_node_stop.py

```diff
@@ -118,7 +118,7 @@ def timer_callback(self):
             self.set_emergency_stop()
             if not self.nogo_pre_active :
                 self.update_service_list()
-                self.set_free_run(True)
+                self.set_free_run(False)
                 self.start_stop_scheduler(False)
```

Comment:
Also be careful with setting free run to True close to stairs, and in places where there is a slight ramp. I stopped doing free runs in monitored navigation before people pressed something on the screen, because I think bad things can happen if you just go to free run without anyone around